experiments/ClimaEarth/components/atmosphere/climaatmos.jl

```diff
@@ -659,7 +670,25 @@ function get_atmos_config_dict(coupler_config::Dict, atmos_output_dir)
         atmos_config["restart_file"] =
             replace(atmos_config["restart_file"], "active" => "0000")
     end
-    return CA.AtmosConfig(atmos_config)
+
+    # Construct the AtmosConfig object
+    toml_dict = coupled_param_dict
```

Comment:
I'm a bit confused by this toml_dict redefinition, do we not want to use the existing `toml_dict`?

Reply:
Oops yes you are right